.travis.yml

```diff
@@ -8,120 +8,147 @@ stages:
   - build
   - test
 
+templates:
```

Comment:
Assuming I've understood correctly, could you leave a little comment to say that this tag is our invention to reuse these config in jobs.include.